pkg/rotator/rotator.go

```diff
@@ -339,12 +343,9 @@ func buildArtifactsFromSecret(secret *corev1.Secret) (*KeyPairArtifacts, error)
 	}, nil
 }
 
-// createCACert creates the self-signed CA cert and private key that will
+// CreateCACert creates the self-signed CA cert and private key that will
```

Comment:
I don't _think_ this method uses anything from `CertRotator`, so should we make this a free-standing function so you don't have to create a rotator to use it? (Ditto for the other method)

Reply:
The cert generation requires: CAName and CAOrganization.
https://github.com/open-policy-agent/cert-controller/blob/6b649a1dbadcbd02af3c5d75cddb49af5950383a/pkg/rotator/rotator.go#L348-L355

Reply:
Ah sorry. Thanks.